reviewpad.yml

```diff
@@ -0,0 +1,71 @@
+api-version: reviewpad.com/v3.x
+
+labels:
+  size/xs:
+    color: '5D9801'
+    description: 'Denotes a change 0-9 lines'
+  size/S:
+    color: '#5D9801'
+    description: 'Denotes a change 10-29 lines'
+  size/M:
+    color: '7F7203'
+    description: 'Denotes a change 30-99 lines'
+  size/L:
+    color: 'A14C05'
+    description: 'Denotes a change 100-499 lines'
+  size/XL:
+    color: 'C32607'
+    description: 'Denotes a change 500-999 lines'
+  size/XXL:
+    color: 'E50009'
+    description: 'Denotes a change 1000+ lines'
+
+workflows:
+  - name: label-pull-request-with-size
+    if:
+      - rule: $size() <= 9
+        extra-actions:
+          - $addLabel("size/XS")
+      - rule: $size() => 10 && $size() <= 29
+        extra-actions:
+          - $addLabel("size/S")
+      - rule: $size() => 20 && $size() <= 99
+        extra-actions:
+          - $addLabel("size/M")
+      - rule: $size() => 100 && $size() <= 499
+        extra-actions:
+          - $addLabel("size/L")
+      - rule: $size() => 500 && $size() <= 999
+        extra-actions:
+          - $addLabel("size/XL")
+      - rule: $size() >= 1000
+        extra-actions:
+          - $addLabel("size/XXL")
+
+  - name: empty-description
+    on:
+      - 'issue'
+      - 'pull_request'
```

Comment:
This configuration does not happen at the reviewpad configuration level but at the github action configuration level. More specifically [here](https://github.com/schmelto/Portfolio/blob/main/.github/workflows/reviewpad.yml#L4).

The same is true for line [56](https://github.com/schmelto/Portfolio/pull/224/files#diff-d7a202cc076357dca8f9a8931b252f8ba4803ea0d12b140e62bea228410ad634R56) and [65](https://github.com/schmelto/Portfolio/pull/224/files#diff-d7a202cc076357dca8f9a8931b252f8ba4803ea0d12b140e62bea228410ad634R65).

I understand why you did that. I guess that having this kind of configuration (where the rule should run, if `pull-request` or `issue`) is useful. I would create an issue for this on Reviewpad project. Please let us know what you think 😄

Reply:
Yeah it would be nice!

Reply:
Hi @schmelto It turns out that you're 100% right. Totally miss that `on` feature 🤦‍♂️ . It works exactly has you have it in the pr. `issue` will run the workflow only on issues, `pull_request` only on pull requestes. Having both run for both issues and pull request. When no value is provided then it only runs for pull requests. Sorry for misguided you.